python/lsst/synpipe/positionGalSimFakes.py

```diff
@@ -103,6 +101,9 @@ def run(self, exposure, background):
             except KeyError:
                 raise KeyError("No RA/DEC column in {} table".format(self.config.galList))
 
+            skyToPixelMatrix = wcs.linearizeSkyToPixel(galCoord, lsst.afw.geom.degrees)
```

Comment:
add a comment about why the 3600 is necessary (I agree it is, but magic numbers should not be used without some documentation

Reply:
The 3600.0 is kept from the previous version of this code the documentation of which is out of the scope of this ticket.